elasticsearch/src/test/java/org/hibernate/search/elasticsearch/testutil/OrmTestRunner.java

```diff
@@ -20,7 +20,7 @@
  * @author Gunnar Morling
  */
 @RunWith(ClasspathSuite.class)
-@ClasspathSuite.ClassnameFilters({ ".*ProjectionQueryTest" })
-// @ClasspathSuite.ClassnameFilters({ "org.hibernate.search.test.*" })
+//@ClasspathSuite.ClassnameFilters({ ".*ProjectionQueryTest" })
+ @ClasspathSuite.ClassnameFilters({ "org.hibernate.search.test.jpa.SpatialQueryingJPATest" })
 public class OrmTestRunner {
```

Comment:
Yes, this was a mistake. Fortunately, this broke the build.
I'll make sure to only add the "Ready for review" tag when the build passes, in the future... Sorry for the inconvenience.